cookbook/upgrade/_update_all_packages.rst.inc

```diff
@@ -0,0 +1,16 @@
+You may also want to upgrade the rest of your libraries. If you've done a
+good job with your `version constraints`_ in ``composer.json``, you can do
+this safely by running:
+
+.. code-block:: bash
+
+    $ composer update
+
+.. caution::
+
+    Beware, if you have some bad `version constraints`_ in your
```

Comment:
I think bad can be replaced with something better, "risky", "unspecific", "nonrestrictive" or "less specific".
To give a better idea why they are bad.